src/makeCalculator.js

```diff
@@ -37,7 +37,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+
+    add: function(number) {
```

Comment:
it should be object field 
```suggestion
  

  const calculator = {
    result: 0,
    
    add: function(number) {
```